build.sbt

```diff
@@ -134,6 +134,7 @@ inThisBuild(
     ),
     testFrameworks := List(),
     resolvers += Resolver.sonatypeRepo("public"),
+    resolvers += Resolver.sonatypeRepo("snapshot"),
```

Comment:
I know this is already merged, but does the snapshot resolver need to stay in here?

Reply:
It's useful for testing out snapshot releases. We can remove it for sure, but it shouldn't matter too much.